dozer/cogs/music.py

```diff
@@ -477,6 +477,24 @@ async def connect(self, ctx: commands.Context, *, channel: discord.VoiceChannel
     `{prefix}connect` - connects the music module to the channel you're in
     """
 
+    @command()
+    async def removefromqueue(self, ctx: commands.Context, *, position: int):
+        """Connect to a voice channel."""
```

Comment:
gonna assume this does something different than connecting to a voice channel

Reply:
oops